staging/src/k8s.io/kubectl/pkg/cmd/diff/diff.go

```diff
@@ -94,8 +94,8 @@ const (
 
 // diffError returns the ExitError if the status code is less than 1,
 // nil otherwise.
-func diffError(err error) exec.ExitError {
-	if err, ok := err.(exec.ExitError); ok && err.ExitStatus() <= 1 {
+func diffError(err error) *exec.ExitError {
+	if err, ok := err.(*exec.ExitError); ok && err.ExitCode() <= 1 {
```

Comment:
I think this is safe to do? It's probably more system independent than the previous wrapper.

https://github.com/kubernetes/utils/blob/3a6ce19ff2f91f3dc49243f90f7834a0f1e4aaf3/exec/exec.go#L218-L225